tmail-backend/jmap/extensions/src/main/scala/com/linagora/tmail/james/jmap/settings/JmapSettings.scala

```diff
@@ -50,21 +50,21 @@ case class JmapSettingsValue(value: String) extends AnyVal
 trait JmapSettingEntry
 
 object JmapSettings {
-  val trashCleanupEnabledSetting: JmapSettingsKey = JmapSettingsKey.liftOrThrow("trash.cleanup.enabled")
-  val trashCleanupPeriodSetting: JmapSettingsKey = JmapSettingsKey.liftOrThrow("trash.cleanup.period")
+  val TrashCleanupEnabledSetting: JmapSettingsKey = JmapSettingsKey.liftOrThrow("trash.cleanup.enabled")
+  val TrashCleanupPeriodSetting: JmapSettingsKey = JmapSettingsKey.liftOrThrow("trash.cleanup.period")
 
-  val weeklyPeriod: String = "weekly"
-  val monthlyPeriod: String = "monthly"
+  val WeeklyPeriod: String = "weekly"
+  val MonthlyPeriod: String = "monthly"
```

Comment:
I don't think this format is ok for static scala variables? Realy don't think it should start with a upper case. That way is for classes/objects names

Reply:
FYI I think previous format was ok. 

Static variables in scala objects in our code we have two ways I can see:
- camel case, like previous format
- everything in upper case

One or the other is fine IMO